### ISSUE TITLE
ARM support in dotCMS

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#19192]
With the rise of the ARM architecture, I would like to see dotCMS fully support running on ARM based machines, such as AWS's Graviton2.  Supporting ARM fully would make dotCMS much more cost effective to run.

In testing dotCMS on a Graviton2 instance, I had to do the following to get it to run - change the `java.security` to use the non-blocking random provider, e.g.
`vi /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.252.b09-2.amzn2.0.1.aarch64/jre/lib/security/java.security` and change these lines:

```
securerandom.source=file:/dev/urandom
securerandom.strongAlgorithms=NativePRNGNonBlocking:SUN
```

### Problems:

In running dotCMS, we have two main issues we need to address.

1. Sass won't compile - We need to update the libsass and jsass libraries to support the ARM architecture.  I can confirm it works if we update our jsass library to version: '5.10.3'

1. WebP won't render - we get these errors:

```
Caused by: java.lang.UnsatisfiedLinkError: com.luciad.imageio.webp.WebPEncoderOptions.createConfig()J
	at com.luciad.imageio.webp.WebPEncoderOptions.createConfig(Native Method) ~[dot.webp-imageio-core-0.1.4.jar:?]
	at com.luciad.imageio.webp.WebPEncoderOptions.<init>(WebPEncoderOptions.java:26) ~[dot.webp-imageio-core-0.1.4.jar:?]
	at com.luciad.imageio.webp.WebPWriteParam.<init>(WebPWriteParam.java:30) ~[dot.webp-imageio-core-0.1.4.jar:?]
	at com.dotmarketing.image.filter.WebPImageFilter.runFilter(WebPImageFilter.java:40) ~[dotcms_5.3.7_e69b2fd.jar:?]
```

This is because we do not ship our webp support with the native libraries needed to run on ARM.  You can what we ship with here:

https://github.com/dotCMS/webp-imageio-core/tree/master/src/main/resources/natives

We need to update that to include the webp arm native libs

**Comment by wezell** [comment]
This has been completed

**Comment by Zakinator123** [comment]
Does that backport commit mean that ARM support is available in DotCMS 5.3.8.5?
